samples/wifi/shell/README.rst

```diff
@@ -156,6 +156,34 @@ Supported CLI commands
      - | Configure Wi-Fi power save for wakeup mode
        | dtim - Wakeup mode for the DTIM interval
        | listen_interval - Wakeup mode for the Listen interval
+   * - mode
+     - | This command may be used to set the Wi-Fi device into a specific mode of operation
+       | parameters:
+       | [-i : Interface index - optional argument
```

Comment:
Is the unbalanced square bracket expected?

Reply:
following the practice that was follwed for other commands in wifi_shell.c from which this help is taken

Reply:
No, it's an overlook, `wifi scan` has optional args in `[]`, so, we should fix the upstream shell help as well.

Reply:
okay :)

Reply:
done